packaging/arch/PKGBUILD

```diff
@@ -14,19 +15,25 @@ checkdepends=('python' 'squashfs-tools' 'indent' 'shellcheck')
 
 options=('!strip' 'emptydirs')
 install=snapd.install
-source=("git+https://github.com/snapcore/$pkgname.git")
+source=("git+https://github.com/bboozzoo/$pkgbase.git")
 md5sums=('SKIP')
 
-_gourl=github.com/snapcore/snapd
+pkgdesc="Service and tools for management of snap packages."
+depends=('squashfs-tools' 'libseccomp' 'libsystemd')
+provides=($pkgbase)
+# Community package is split into snapd and snap-confine, make sure we replace
+# both bits
+conflicts=($pkgbase 'snap-confine')
```

Comment:
IIRC, Arch PKGBUILD has something like `Obsoletes`... I think it's called `replaces`? You probably want that with `conflicts`.

Reply:
Looking at the spec it seems you are right: https://wiki.archlinux.org/index.php/PKGBUILD#replaces

Reply:
Depends on how you look at it, but IMO https://wiki.archlinux.org/index.php/PKGBUILD#conflicts is more applicable. Especially since we're providing an alternative version of a package that is in the repo. Actually the note on `replaces` has this:

> If providing an alternate version of an already existing package or uploading to the AUR, use the conflicts and provides arrays, which are only evaluated when actually installing the conflicting package.